src/sagemaker/jumpstart/factory/model.py

```diff
@@ -505,6 +510,91 @@ def get_deploy_kwargs(
     return deploy_kwargs
 
 
+def get_register_kwargs(
+    model_id: str,
+    model_version: Optional[str] = None,
+    region: Optional[str] = None,
+    tolerate_deprecated_model: Optional[bool] = None,
+    tolerate_vulnerable_model: Optional[bool] = None,
+    sagemaker_session: Optional[Any] = None,
+    supported_content_types: List[str] = None,
+    response_types: List[str] = None,
+    inference_instances: Optional[List[str]] = None,
+    transform_instances: Optional[List[str]] = None,
+    model_package_name: Optional[str] = None,
+    model_package_group_name: Optional[str] = None,
+    image_uri: Optional[str] = None,
+    model_metrics: Optional[ModelMetrics] = None,
+    metadata_properties: Optional[MetadataProperties] = None,
+    marketplace_cert: bool = False,
```

Comment:
Again we do not need this because we do not want to support marketplace registration